src/services/add-query-to-queue.ts

```diff
@@ -167,4 +181,67 @@ export default class AddQueryToQueue {
       await interaction.editReply(`u betcha, **${firstSong.title}** and ${newSongs.length - 1} other songs were added to the queue${extraMsg}`);
     }
   }
+
+  private async enrichWithSkipSegments(songs: SongMetadata[]) {
+    const promises = songs.map(async song => {
+      if (!this.sbClient
+          || this.sbClientTimeout
+          || song.source !== MediaSource.Youtube
+          || !song.url) {
+        return song;
+      }
+
+      try {
+        return await this.sbClient.getSegments(song.url, ['music_offtopic']).then(segments => {
```

Comment:
let's do 
```suggestion
        const segments = await this.sbClient.getSegments(song.url, ['music_offtopic'])
```
to remove an indentation level